csgo_internal/csgo_internal/AntiAim.cpp

```diff
@@ -25,6 +25,16 @@ void CAntiAim::Run()
 	if( G::LocalPlayer->GetMoveType() == MOVETYPE_LADDER || G::LocalPlayer->GetMoveType() == MOVETYPE_NOCLIP )
 		return;
 
+	auto pWeapon = G::LocalPlayer->GetWeapon();
+	if (!pWeapon->GetWeaponTypewitham()) //如果返回FALSE
```

Comment:
also when pulling out knife /etc you will freeze up and not able to move until AA is turned off so this is basically useless